shootout/src/bench.rs

```diff
@@ -12,12 +12,13 @@ use zero_knowledge::ZeroKnowledge;
 
 // Please remove this when a better way is found.
 #[derive(Clone)]
-pub enum ZKP {
+pub enum ZKP<'a> {
```

Comment:
remove any generics, this code should not be touching the structure of the main crate, other than adding the Vampir backend